pkgs/applications/science/logic/cvc5/default.nix

```diff
@@ -1,4 +1,6 @@
-{ lib, stdenv, fetchFromGitHub, pkg-config, cmake, flex, cadical, symfpu, gmp, python3, gtest, libantlr3c, antlr3_4, boost, jdk }:
+{ lib, stdenv, fetchFromGitHub, pkg-config, cmake, flex, cadical, symfpu, gmp, python3, gtest, libantlr3c, antlr3_4, boost, jdk, pythonBindings ? false }:
+
+with lib;
```

Comment:
[Never use global `with lib;`.](https://github.com/NixOS/nixpkgs/issues/208242)

Reply:
Oh, I see -- that makes sense in retrospect.  I'll adjust it to not use `with`. 

Could you elaborate on the issue with the diffs? I can reformat it while I'm at it but I don't understand which diffs we're talking about.

Reply:
Having all the dependencies on one line makes the diff appear as if the entire line has deleted and added back again, whereas in reality only `pythonBindings` has been added. 

This is why modern formatting styles put a newline between every dependency, so that additions like these are much easier to read. [Example](https://github.com/pluiedev/nixpkgs/blob/f7c5d7966670a65b45306aecf7eb209bbb1686fb/pkgs/by-name/nz/nzp-fteqw/package.nix#L1-L40)